### DISCUSSION TITLE
Dịch closed-form solution

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#61] [Terminology]
Từ này mọi người đều hiểu là bài toán có nghiệm có thể tính được trực tiếp. Hiện tại ở nhiều phần đã dịch là `dạng đóng` (`phương trình dạng đóng`, `nghiệm dạng đóng`).
Anh @ngcthuong có đề xuất là `nghiệm đúng` (ghép context vào sẽ là `phương trình có nghiệm đúng`, `nghiệm đúng`).
Mn nghĩ sao?
cc: @mlbvn/handson-ml2

**Comment by lkhphuc** [comment]
Mình thấy từ này được dịch là `biểu thức dạng đóng` khá phổ biến rồi nên không ủng hộ lắm. 
`Nghiệm đúng` nên để dành cho `exact solution`. https://mathworld.wolfram.com/ExactSolution.html
> Exact Solution
>
>As used in physics, the term "exact" generally refers to a solution that captures the entire physics and mathematics of a problem as opposed to one that is approximate, perturbative, etc. Exact solutions therefore need not be closed-form.

**Reply by ngcthuong** [reply]
Phương trình nghiệm đúng là giải chính xác được bằng công thức toán học nó trái ngược với các giải xấp xỉ "gần đúng".

**Reply by rootonchair** [reply]
Em nghĩ nếu từ này trong toán có rồi thì mình nên sử dụng ạ

**Reply by ngcthuong** [reply]
Ý mình là em có dẫn chứng ở sách nào dịch vậy không vì mình không rõ nên mình mới đặt vấn đề này. Dẫn chứng ở trên là tiếng ANh và thuộc bên vật lý.

**Reply by rootonchair** [reply]
Vậy anh @lkhphuc có thể trích nguồn được sử dụng cho từ này không anh?

**Reply by ngcthuong** [reply]
Nhân tiện nhóm mình có ai ở Vn hay ở đâu có thể tham khảo mấy cuốn sách toán cơ bản này nhỉ? Bên toán chắc học chuyên hơn nên rõ từ khóa hơn. Mình xưa không học đến/hoặc không chú ý mấy cái này nên không rõ lắm.

**Reply by rootonchair** [reply]
Nếu có tên môn thì em có thể kiếm ạ

**Reply by ngcthuong** [reply]
Để mình thử hỏi mấy anh bên toán coi sao.

**Reply by thanhcsf** [reply]
Dạ anh @ngcthuong ơi, từ này có thảo luận gì mới không ạ?
